src/Engine/lib/api/routers/SettingsRouter.py

```
@@ -81,38 +89,44 @@ def clear_login_data() -> JSONResponse:
-        
+
         return JSONResponse(
             status_code=200,
             content={
                 "success": True,
                 "message": "All login data cleared successfully",
             },
         )
-    
+
     except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)